tests/utilities/parsing.py

```diff
@@ -64,6 +78,10 @@ def test_lightning_getattr(tmpdir):
         value = lightning_getattr(m, 'learning_rate')
         assert value == i, 'attribute not correctly extracted'
 
+    model5 = models[-1]
```

Comment:
```suggestion
    model5 = models[4]
```
or it will potentially break if a new model is added.